src/routers/handlers/index/home.ts

```diff
@@ -11,13 +13,19 @@ export class HomeHandler extends GenericHandler<BaseViewData> {
         return {
             ...baseViewData,
             title: "Apply to file with Companies House using software",
-            backURL: null
+            backURL: null,
         };
     }
 
-    public execute (req: Request, _response: Response): ViewModel<BaseViewData> {
+    public execute (req: Request, _response: Response): ViewModel<BaseViewData> | Redirect {
         logger.info(`GET request to serve home page`);
-        // ...process request here and return data for the view
+        var temp = env.FEATURE_FLAG_GDS_START_PAGE_290424;
```

Comment:
This needs to be removed now.
It was only for viewing the variable for debugging.
```suggestion
```

Reply:
done